estate/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+{
+    'name': "estate app",
+
+    'summary': """
+        estate module"
+    """,
+
+    'description': """
+        estate module"
```

Comment:
```suggestion
Estate module
```
Describe your module in the description to provide a brief overview in one or two sentences.